internal/store/migrations.go

```diff
@@ -79,4 +81,97 @@ var migrations = []migration{
 			return err
 		},
 	},
+	{semver.MustParse("0.2.0"), semver.MustParse("0.3.0"),
+		func(e execer) error {
+			m := func(i int64) int64 {
+				if i == 0 {
+					return 0
+				}
+				return i / 1000
+			}
+
+			uploadRows, err := e.Query(`SELECT ID, CreateAt, CompleteAt FROM upload;`)
```

Comment:
I think we might be able to do it a little simpler with just SQL, as we want to update basically all time columns, for example:
```SQL
UPDATE upload SET createat = createat / 1000, completeat = completeat / 1000;
```
It might be worth it for the sake of simplicity. What do you think?

Reply:
Thanks, I knew there was a better way to do this, but was worried about possible issues with 0 values and division. This works like a charm though. 👍